README.adoc

```diff
@@ -108,41 +114,40 @@ their explanation:
 1.  `vagrant service-manager env [service] [--script-readable]`
 +
 Displays connection information for all active services in the box in a
-manner that can be evaluated in a shell. If a `service` is specified,
-only the information for that service is displayed. When
-`--script-readable` is specified the output is in `key=value` format.
-The supported services are: Docker, OpenShift.
-+
-*Note:* If a `service` is not specified, it displays information of all active services.
+manner that can be evaluated in a shell. +
+If a `service` is specified, only the information for that service is displayed.
+The supported services are: Docker, OpenShift. +
+When `--script-readable` is specified the output is in `key=value` format.
 
-1.  `vagrant service-manager box [command]`
+1.  `vagrant service-manager box [command] [--script-readable]`
 +
 Displays box related information like release version, IP etc.
-
-1.  `vagrant service-manager box version [--script-readable]`
-+
-Displays the version and release information of the running VM. When
-`--script-readable` is specified the output is in `key=value` format.
-
-1.  `vagrant service-manager box ip`
 +
-Displays the routable IP address of the running VM.
+--
+The possible options for `command` are:
 
-1.  `vagrant service-manager status [service]`
+* `version`: Displays the version and release information of the running VM.
+* `ip`: Displays the routable IP address of the running VM.
 +
-Lists services and their running state. If a `service` is specified only
-the status of that service is displayed. If no service is provided then
-only supported orchestrators are reported.
+--
+When `--script-readable` is specified the output is in `key=value` format.
 
 1.  `vagrant service-manager [operation] [service]`
 +
-where `operation` could be:
+Displays the status of the services.
```

Comment:
"Manages the life cycle of a service" instead of  "Displays the status of services."